Kiosk/Sale Artwork Details/SaleArtworkDetailsViewController.swift

```diff
@@ -401,11 +401,11 @@ class SaleArtworkDetailsViewController: UIViewController {
                 .subscribeNext { [weak self] blurb in
                     guard let me = self else { return }
 
-                    let aboutArtistHeaderLabel = label(.Header)
+                    let aboutArtistHeaderLabel = me.label(.Header)
                     aboutArtistHeaderLabel.text = "About \(artist.name)"
                     me.additionalDetailScrollView.stackView.addSubview(aboutArtistHeaderLabel, withTopMargin: "22", sideMargin: "40")
 
-                    let aboutAristLabel = label(.Body, layout: me.layoutSubviews)
+                    let aboutAristLabel = me.label(.Body, layout: me.layoutSubviews)
```

Comment:
Ah, I see... interesting. I would have thought the compiler would've caught those.